actions/lib/action.py

```diff
@@ -44,11 +46,6 @@ def __init__(self, config):
         secret_access_key = config.get('aws_secret_access_key', None)
         region = config.get('region', None)
 
-        if access_key_id == "None":
-            access_key_id = None
-        if secret_access_key == "None":
-            secret_access_key = None
-
```

Comment:
Why are we removing these lines?  Seems like this will introduce regression for cases where someone passes in a literal "None" value instead of null/None type.

Reply:
They must not be removed, I have removed them unintentionally when cleaning up the code, not realising they had been there from the beginning (I use quotes in my case which are interpreted as missing field and from the next lines, the fields' values become None).

Reply:
Fixed in the most recent iteration. Resolving.